packages/dds/tree/src/codec/codec.ts

```diff
@@ -457,19 +452,21 @@ export enum FluidClientVersion {
 	/** Fluid Framework Client 2.52 and newer. */
 	// New formats introduced in 2.52:
 	// - DetachedFieldIndex FormatV2
-	v2_52 = 2.052,
+	v2_52: "2.52.0" as MinimumVersionForCollab,
 
 	/**
 	 * Enable unreleased and unfinished features.
 	 * @remarks
+	 * pkgVersion will always be the latest version, therefore the version that enables all features.
```

Comment:
This comment could be in privateRemarks, but it should not be un the user facing documentation since the fact that it uses pkgVersion is an implementation detail.

Also note that using pkgVersion here doesn't actually work as documented, since that would only opt into things that are planned to stabilize in the next release.

I think for not it might be best to simply remove the EnableUnstableFeatures option as there isn't a good way to support it with MinimumVersionForCollab